test/index.js

```diff
@@ -19,6 +19,9 @@ const MOCK_ENCRYPTION_DATA = `{"data":"2fOOPRKClNrisB+tmqIcETyZvDuL2iIR1Hr1nO7XZ
 const walletOneSeedWords =
   'puzzle seed penalty soldier say clay field arctic metal hen cage runway';
 const walletOneAddresses = ['0xef35ca8ebb9669a35c31b5f6f249a9941a812ac1'];
+const walletOnePrivateAddress = [
```

Comment:
Nit: this seems to be a private key, not a private "address". Maybe worth renaming this variable to `walletonePrivateKey`.

Reply:
Nit: maybe this should be plural since it's an array? I had made it singular at first as well in my comment, but then reconsidered and edited the comment to make ti plural.